core/lib/l1_contract_interface/src/multicall3/mod.rs

```diff
@@ -7,6 +7,7 @@ use zksync_types::{
 };
 
 /// Multicall3 contract aggregate method input vector struct.
+#[derive(Debug)]
```

Comment:
It is done for tests?

Reply:
Yes. Can the derived `Debug` representation contain confidential data?

Reply:
Don't think so